recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "2.4.rc3" %}
+{% set source_tag = "2.4-rc3" %}
+
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/{{ source_tag }}.tar.gz
+  sha256: 48a52af8f9da6deef5c01d26a30ba4bdc19929999d6680dab6461b92b5a63bd7
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+  features:
+    - vc14  # [win]
```

Comment:
Would drop the `vc14` feature. It is not longer needed with the new compilers.

Though would update this since `openblas` is being linked to based on [these docs]( https://conda-forge.org/docs/blas.html ). A good example is `numpy`. See [these lines]( https://github.com/conda-forge/numpy-feedstock/blob/7695c3eab8623946c187e1d1cd4683b285c9085d/recipe/meta.yaml#L2-L5 ) and [these lines]( https://github.com/conda-forge/numpy-feedstock/blob/7695c3eab8623946c187e1d1cd4683b285c9085d/recipe/meta.yaml#L18-L19 ). Also needs to depends on [`blas`]( https://github.com/conda-forge/numpy-feedstock/blob/7695c3eab8623946c187e1d1cd4683b285c9085d/recipe/meta.yaml#L27 ) in `host` and `run`.